uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/dom/single/impl/BaseSingletonDOMElementFactory.java

```diff
@@ -74,7 +74,11 @@ public void execute() {
 
     @Override
     public void destroyResources() {
-        flush();
+        if (e != null) {
```

Comment:
`destroyResources()` will also be called when, for example, resizing a column. 

This will now cancel the change. Is this the expected behaviour?

Reply:
Well, I am am not aware of specification how such case should behave, should I ask UX team? The problem of these `destroyResources` and `flush` I touched is in they repetition, will start a conversation ,on email probably, where will try to summarize my findings.

Reply:
OK. It is probably something for UX to determine.

I just mentioned it really to highlight `destroyResources()` is not used solely by keyboard operations.

Reply:
@manstis yes, however still think we should have this behavior implemented:
- destroyResources: don't care about value in widget
- flush: store value in widget

I mean, if we want to keep current behavior, I rather vote fore replacing current `destroyResources` usages by `flush`.

Reply:
@jomarko I don't have any issues with this PR.. I was just highlighting. I tend to agree, that use of `<Enter>` or `<Tab>` to _flush_ the value is correct and that resizing a column should _discard_ the change.. green light on this PR from me.

Reply:
@manstis seems Liz have different opinion. She said values are expected to remain if column resized. I need to update the PR.